_sass/_type.scss

```diff
@@ -110,3 +110,31 @@ h4 {
     color: $dark-blue;
   }
 }
+
+.awards-list {
+  list-style-position: outside;
+  
+  li {
+    margin-bottom: 0.5rem;
+    line-height: 1.3;
+
+    &::marker {
+      color: rgba(251,147,6,0.75);
+    }
+  }
+
+  a:link,
+  a:visited {
+    display: inline;
+    text-decoration: underline;
+    text-decoration-color: rgba(251,147,6,0.75);
+    text-decoration-thickness: 5px;
+    -webkit-text-decoration-skip: unset; 
+    text-decoration-skip: unset; 
+    text-decoration-skip-ink: none;
+
+    &::before {
+      display: none;
```

Comment:
What is this for? I don't see any `::before` elements being defined for these links to be hidden.

Reply:
It hides the base link styling, if its not hidden, it breaks when a link spans two lines.